email.android.js

```diff
@@ -8,9 +8,9 @@ var fs = require("file-system");
 exports.available = function () {
   return new Promise(function (resolve, reject) {
     try {
-      var uri = android.net.Uri.fromParts("mailto", "eddyverbruggen@gmail.com", null);
+      var uri = android.net.Uri.fromParts("mailto", "frontend-licenties@essent.nl", null);
```

Comment:
I would change this to something not connected to a real essent address, something like john@doe.com.